jsapp/js/account/usage/yourPlan.component.tsx

```diff
@@ -69,6 +73,10 @@ export const YourPlan = () => {
     }
   }, [env.isReady, subscriptions.isInitialised]);
 
+  const showPlanUpdateLink = areMmosEnabled
```

Comment:
Thinking about this... it would help us to have the organization utils as a hook, even encapsulating the `useOrganizationQuery` and the feature flag verification so that we could have the conditions named as in:

``` typescript
const {organizationData, isOwner} = useOrganization()
```
WDYT?